xml/tuning-ceph-tuning.xml

```diff
@@ -415,7 +415,7 @@ echo {bytes} > /sys/block/rbd0/queue/read_ahead_kb
        utilizing very low-latency media, such as NVMe, for the cephfs metadata pool.
      </para>
      <para>
-       Utlizing the <command>ceph-daemon perf dump</command> command, there is a significant
+       Utlizing the <command>ceph daemon perf dump</command> command, there is a significant
```

Comment:
```suggestion
       Utilizing the <command>ceph daemon perf dump</command> command, there is a significant
```

Reply:
Personally, I'm a bit fan of utlizing but suit yourself